lib/galaxy/tools/toolbox/base.py

```diff
@@ -839,8 +839,7 @@ def quick_load(tool_file, async_load=True):
             if os.path.isdir(child_path) and recursive:
                 self.__watch_directory(child_path, elems, integrated_elems, load_panel_dict, recursive)
             elif self._looks_like_a_tool(child_path):
-                quick_load(child_path, async_load=False)
-                tool_loaded = True
+                tool_loaded = quick_load(child_path, async_load=False)
```

Comment:
Explicit better than implicit?
```suggestion
				tool_id = quick_load(child_path, async_load=False)
                tool_loaded = bool(tool_id)
```

Reply:
Yep, reads better.